components/tyk-gateway/templates/pdb-gw.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.gateway.minAvailable .Values.gateway.maxUnavailable }}
+{{- print "If gateway.minAvailable and gateway.maxUnavailable are set at the same time, minAvailable takes precedence." }}
```

Comment:
Do we need this print statement?

Reply:
I added it as an additional precaution to those that don't read comments in `values.yaml`, since this condition will only be true if someone sets both values to non-zero values at the same time. But if you feel like that's going too far I can remove it.

Reply:
We can keep it. Thank you for explanation.